src/core/Table/Table.tsx

```diff
@@ -88,6 +91,24 @@ export type TableProps<
    * @default 300
    */
   intersectionMargin?: number;
+  /**
+   * A function that will be used for rendering a component for each row if that row is expanded.
+   * Component will be placed right after the row, in the same row group div.
+   */
+  subComponent?: (row: Row<T>) => React.ReactNode;
```

Comment:
Not sure whether pass row or row data (`row.original`)

Reply:
I think `row` could be quite useful since it gives much more information about the row state. I can even do `row.toggleRowExpanded()` inside subcomponent for example.